mods/gennext/README.md

```diff
@@ -145,7 +145,7 @@ Major changes:
 
 - Twister is now a 80 base power Flying move with a 30% confusion chance
 
-- Floette-Eternal-Flower and Volcanion are released.
+- Floette-Eternal-Flower are released.
```

Comment:
Grammar nit: Floette-Eternal-Flower is released.

Reply:
Ill fix this

Reply:
...I already fixed this

Reply:
I noticed after replying